README.md

```diff
@@ -177,7 +188,7 @@ For more information on building, running, and developing on the React-based UI,
 
 ## More information
 
-* Godoc documentation is available via [pkg.go.dev](https://pkg.go.dev/github.com/prometheus/prometheus). Due to peculiarities of Go Modules, v2.x.y will be displayed as v0.x.y.
+* Godoc documentation is available via [pkg.go.dev](https://pkg.go.dev/github.com/prometheus/prometheus). Due to peculiarities of Go Modules, v3.x.y will be displayed as v0.3x.y, while v2.x.y will be displayed as v0.x.y.
```

Comment:
Again, we should mention that `x` is always padded to two digits, with a leading zero if needed.

Reply:
Updated in https://github.com/prometheus/prometheus/pull/15361/commits/9f4f08d0d50f6322c947646b271d8d1a5eb56807, also updated `x.y` to `y.z` to be consistent with earlier explanation